jarbas/dashboard/admin.py

```diff
@@ -74,6 +88,40 @@ def queryset(self, request, queryset):
         return queryset.suspicions() if self.value() == 'yes' else queryset
 
 
+class MonthListFilter(SimpleListFilter):
+
+    title = 'mês'
+    parameter_name = 'month'
+    options = ((k, v) for k, v in MONTHS.items())
```

Comment:
This should be something like `options = tuple((k, v) for k, v in MONTHS.items())` so it is displayed on the dashboard. Right now this filter is not being shown ;)